File: task14.py

dict={
}
dict["Wheel"]=4
dict["Motor"]=2

dict["Sensor"]=5

dict["Wheel"]-=1
print(dict["Wheel"])
dict["Motor"]+=3
print(dict["Motor"])
print(dict.get("laser",0)) # returns 0 if it doesnt exist 

File: Team_Workspace/Fatma_Ahmed/cpp_tasks/task14/task14.cpp
// task14.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include <iostream>

#include <map>

using namespace std;


int main()
{
	map<string, int>test_map;
	test_map.insert({ "Wheel",4 });
	test_map.insert({ "Motor",2 });
	test_map.insert( { "Sensor",5 });


	test_map["Wheel"]--;
	test_map["Motors"] += 3;


	cout << test_map["Laser"]<<endl;
	cout << test_map["Wheel"]<<endl;
	cout << test_map["Motor"]<<endl;
	cout << test_map["Sensor"]<<endl;






}

